core/src/main/scala/kafka/zookeeper/ZooKeeperClient.scala

```diff
@@ -569,6 +601,11 @@ sealed abstract class AsyncResponse {
   def metadata: ResponseMetadata
 }
 
+sealed trait GetChildrenResponseNoStat extends AsyncResponse {
```

Comment:
Why do we want to customize to a NoStat?

Reply:
The paginated call doesn't support the Stat argument, and there are a handful of getChildren() callers that actually do make use of it. (My first thought was to get rid of the argument altogether, but no joy.) Having a shared superclass (or whatever it's called in Scala) saves a little bit of code in getAllTopicsInCluster() in KafkaZkClient.

Reply:
Makes sense.